src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/shared/containers/Presentation.tsx

```diff
@@ -50,11 +52,21 @@ const PresentationComponent = (props: IPresentationProvidedProps) => {
   };
 
   const handleModalCloseButton = () => {
-    const origin = window.location.origin;
-    if (window) {
-      window.location.href = returnUrlToMessagebox(origin, party.partyId);
+    const queryParameterReturnUrl = returnUrlFromQueryParameter();
+    const messageBoxUrl = returnUrlToMessagebox(window.location.origin, party.partyId);
+    console.log('Q: ', queryParameterReturnUrl)
```

Comment:
- [ ]  console.log should be removed

Reply:
But but I like console logging stuff all the time, thank you for noticing, I'll remove these asap